### ISSUE TITLE
REQUEST: 10s count for 2

### ISSUE THREAD

**Issue by Auctorion** [Humblemonk/DiceMaiden#55]
Some systems like Exalted have it so that 10s count for 2 successes rather than 1, rather than having them explode. Would be nice to be able to specify that a 10 counts for 2 successes, or indeed that any specific side should count for a specified number of successes rather than just 1.

**Comment by aakinp** [comment]
I would like this feature, but for d6s as well. So that 6 counts for 2 successes.

**Comment by Virtueboy** [comment]
> Some systems like Exalted have it so that 10s count for 2 successes rather than 1, rather than having them explode. Would be nice to be able to specify that a 10 counts for 2 successes, or indeed that any specific side should count for a specified number of successes rather than just 1.

Definitely would help our exalted 2nd edition games!

**Comment by NexplorerVA** [comment]
Yeah wod 4th edition would need that feature for specialisations too... would be nice to have that included :)

**Comment by Auctorion** [comment]
> Yeah wod 4th edition would need that feature for specialisations too... would be nice to have that included :)

Where is that in the rules?

**Comment by NexplorerVA** [comment]
> > Yeah wod 4th edition would need that feature for specialisations too... would be nice to have that included :)
> 
> Where is that in the rules?

Mage - The Ascension explains the specialityrules at page 274, Werewolf - The Apocalypse at page 123 (Both 20th Anniversary Editions)... I don't know about Vampire but since these two systems are using these rules I guess Vampire will have that too (instead of the reroll-rule of tens they used up to the 3rd edition)

**Comment by Auctorion** [comment]
> > > Yeah wod 4th edition would need that feature for specialisations too... would be nice to have that included :)
> > 
> > 
> > Where is that in the rules?
> 
> Mage - The Ascension explains the specialityrules at page 274, Werewolf - The Apocalypse at page 123 (Both 20th Anniversary Editions)... I don't know about Vampire but since these two systems are using these rules I guess Vampire will have that too (instead of the reroll-rule of tens they used up to the 3rd edition)
> 
> Edit: Looked it up... Vampire - The Masquerade (20th anniversary edition) page 96, they are using the same rule.

Ah, so it's only in the 20th Anniversary editions. I guess that explains why I couldn't find any mention of it in the Chronicles of Darkness core book.

**Comment by Auctorion** [comment]
@Humblemonk, I think this ticket is easier to implement as you have a very similar code here: https://github.com/Humblemonk/DiceMaiden/issues/16

However I tested it with d10s and it seems not to work. 
![Image](https://user-images.githubusercontent.com/63723816/148692492-c17bccc1-d3dc-4f22-97cc-cf98a9a7652c.jpeg)

**Comment by Onigato** [comment]
It would appear this still isn't implemented yet, I'd like to put my two-cents forward that I'd like this feature, maximum rolled dice counting double successes, please.

**Comment by Humblemonk** [comment]
Going to close this out as this work has been implemented for a few systems now. If any other system needs this for an alias, please create a new issue here https://github.com/Humblemonk/dicemaiden-rs
